### PR TITLE
Docs: Add (rough) sidebar

### PR DESCRIPTION
**Pull request by paulathevalley**
- Extract fonts and color variables to a `settings` file 
- Add `<Sidebar/>` component
- Add `<Docs/>` component with filler text

/cc @david-davidson @coopy 

screencapture-localhost-3000-1448069605904

### CODE REVIEW

File: components/docs.jsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+          </section>
+        </main>
+        <Footer/>
+        <Style rules={theme}/>
+      </div>
+    )
+  }
+}
+
+if (typeof document !== "undefined") {
```
Comment (david-davidson) [inline components/docs.jsx]: I think we're better off *not* having the component render itself now that we've got a router--we can just export it as-is. :)

File: components/theme.js [hunk 2]
```diff
@@ -71,7 +53,7 @@ export default {
     verticalAlign: 'middle'
   },
   'th, td': {
-    border: '1px solid ' + palestSand,
+    border: '1px solid ' + settings.palestSand,
```
Comment (david-davidson) [inline components/theme.js]: `palestSand` => maybe most poetic color variable ever

**Comment by david-davidson** [comment]
This had a merge conflict, since `docs.jsx` changed since you opened the PR (I pushed my far more WIP `docs.jsx` component). I went ahead and merged in `docs`, so you should do a quick pull. :+1:

**Comment by david-davidson** [comment]
:boom: / all gold no mold / :tada:

File: components/sidebar.jsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+      },
+      selectedLink: {
+        color: settings.red,
+        fontWeight: 'bold'
+      },
+      link: {
+        boxShadow: 'none',
+        color: settings.darkSand,
+        fontWeight: 'normal'
+        // ':hover': {
```
Comment (coopy) [inline components/sidebar.jsx]: Remove commented-out code

**Comment by david-davidson** [comment]
@paulathevalley Gonna go ahead and merge this!

**Pull request merged**